modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java

```diff
@@ -70,25 +76,32 @@ public static void init() throws NoSuchFieldException {
         temp.put(RabbitMQStore.HOST_PORT, PORT);
         temp.put(RabbitMQStore.VIRTUAL_HOST, VIRTUAL_HOST);
         temp.put(RabbitMQStore.QUEUE_NAME, QUEUE);
+        temp.put(RabbitMQStore.RETRY_INTERVAL, "0");
+        temp.put(RabbitMQStore.RETRY_COUNT, "0");
         rabbitMQStore.setParameters(temp);
         AxisConfiguration axisConfiguration = new AxisConfiguration();
         configurationContext = new ConfigurationContext(axisConfiguration);
         SynapseConfiguration synapseConfiguration = new SynapseConfiguration();
         Axis2SynapseEnvironment env = new Axis2SynapseEnvironment(configurationContext, synapseConfiguration);
-        rabbitMQStore.init(env);
+        try {
+            rabbitMQStore.init(env);
+        } catch (Exception ignore) {
```

Comment:
What are we trying to assert here? If it's the 'UnknownHostException', let's assert that. catching 'Exception will hide any exception that's thrown, hence make the test case not usefull

Reply:
I tried to do something similar to `@Test(expected = SynapseException.class)`. But `@BeforeClass` annotation doesn't support that. Therefore, catch the exception to make sure init doesn't fail.